Signup/signup.css

```diff
@@ -0,0 +1,22 @@
+
+.signup-heading {
+  text-align: center;
+  padding-bottom: 2.5rem;
```

Comment:
suggestion : For spacings you can use variables for better consistency.

Reply:
I have used variable for 10px,20px and 30px . For larger size i specify it in a class

Reply:
> I have used variable for 10px,20px and 30px . For larger size i specify it in a class

ok I have approved it!